Content.Server/Chat/ChatCommands.cs

```diff
@@ -51,14 +60,17 @@ public void Execute(IConsoleShell shell, IPlayerSession player, string[] args)
 
             var action = string.Join(" ", args);
 
-            chat.EntityMe(player.AttachedEntity, action);
+            // As EntitySay, Hacky way of getting the Player's Mind
+            var mindComponent = player.ContentData().Mind;
+            // Now do the /me as the Player's Mind.
+            chat.EntityMe(mindComponent.OwnedEntity, action);
         }
     }
 
     internal class OOCCommand : IClientCommand
     {
         public string Command => "ooc";
-        public string Description => "Send Out of Character chat messages.";
+        public string Description => "Send Out Of Character chat messages.";
```

Comment:
Updated description to better reflect **O**ut **O**f **C**haracter in the description, more of a preference thing looking back now I guess?